src/Manage/comonent/ManageLookup.js

```diff
@@ -1,10 +0,0 @@
-import React, { Component } from "react";
-import { Text, View } from "react-native";
-
-class ManageLookup extends React.Component {
-  render() {
```

Comment:
Try avoid using `export default` instead prefer exporting then right away. `export class xxx` and prefer import it via destructuring... `import {XX} from '..'`. That way the name of the class is maintained consistently at the consumer end and easy for lookup and refactoring...